API-proposed.yaml

```diff
@@ -3299,12 +3632,64 @@ components:
           format: int64
         style:
           $ref: "#/components/schemas/OutputElementStyle"
-        xValues:
+        xValuesDescription:
           type: array
-          description: Series' X values
+          description: Series' X axis description
           items:
-            type: integer
-            format: int64
+            required:
+            - dataType
+            - label
+            - unit
+            type: object
+            properties:
+              dataType:
+                type: string
+                description: The type of data this axis represents
+                enum:
+                - NUMBER
+                - BINARY_NUMBER
+                - TIMESTAMP
+                - DURATION
+                - STRING
+                - TIME_RANGE
+              unit:
+                type: string
+                description: "Unit associated with this axis (e.g., ns, ms)"
+              label:
+                type: string
+                description: Label for the axis
+              axisDomain:
+                $ref: "#/components/schemas/AxisDomain"
+        yValuesDescription:
+          type: array
```

Comment:
this is not correct, I think. It's not an array. The swagger documentation needs to be updated for that and the API regenerated.